prepare/cards/numeric_nlg.py

```diff
@@ -0,0 +1,56 @@
+from unitxt.blocks import (
+    AddFields,
+    LoadHF,
+    MapHTMLTableToJSON,
+    RenameFields,
+    SerializeTableAsMarkdown,
+    Task,
+    TaskCard,
+    TemplatesList,
+)
+from unitxt.catalog import add_to_catalog
+from unitxt.splitters import SplitRandomMix
+from unitxt.templates import InputOutputTemplate
+from unitxt.test_utils.card import test_card
+
+card = TaskCard(
+    loader=LoadHF(path="kasnerz/numericnlg"),  # TODO: load from github repo
+    preprocess_steps=[
+        SplitRandomMix({"train": "train", "validation": "validation", "test": "test"}),
+        AddFields(fields={"type_of_input": "table", "type_of_output": "text"}),
+        MapHTMLTableToJSON(field_to_field=[["table_html_clean", "table_out"]]),
+        SerializeTableAsMarkdown(field_to_field=[["table_out", "input"]]),
+        RenameFields(field_to_field={"description": "output"}),
+    ],
+    task=Task(
+        inputs={"input": "str", "type_of_input": "str", "type_of_output": "str"},
+        outputs={"output": "str"},
+        prediction_type="str",
+        metrics=[
+            "metrics.bleu",
+            "metrics.rouge",
+            "metrics.bert_score.bert_base_uncased",
+            "metrics.meteor",
+        ],
+        # add PARENT [we leave it for now, as it doesn't have to be the chosen metric]
+        augmentable_inputs=["input"],
+    ),
+    templates=TemplatesList(
+        [
+            InputOutputTemplate(
+                input_format="Given the following {type_of_input}, generate the corresponding {type_of_output}. {type_of_input}: {input}",
+                output_format="{output}",
+                postprocessors=[],
+            ),
+        ]
+    ),
```

Comment:
```suggestion
    templates="templates.generation.all",
```
If this is not good because you dont want the postprocessors I suggest you save new list in the catalog under: `templates.generation.long`

Again the purpose is usability for others and better indexing in the demo. 

If you really dont want to do this (altough reccomnded) please use:

```python 
templates=[
      "templates.generation.default[postprocessors=[]]"
],

Reply:
ok, I think we can live with the current postprocessors in the generation template for now